conduit/data/datasets/vision/camelyon17.py

```diff
@@ -0,0 +1,170 @@
+from enum import Enum, auto
+from pathlib import Path
+from typing import ClassVar, Optional, Union, cast
+
+import pandas as pd
+from ranzen.decorators import enum_name_str, parsable
+from ranzen.misc import str_to_enum
+import torch
+
+from conduit.data.datasets.utils import ImageTform, UrlFileInfo, download_from_url
+from conduit.data.datasets.vision.base import CdtVisionDataset
+
+__all__ = [
+    "Camelyon17",
+    "Camelyon17Split",
+    "Camelyon17SplitScheme",
+]
+
+
+@enum_name_str
+class Camelyon17SplitScheme(Enum):
+    official = auto()
+    """Oficial split."""
+
+    mixed_to_test = auto()
+    """ 
+    For the mixed-to-test setting, slide 23 (corresponding to patient 042, node 3 in the
+    original dataset) is moved from the test set to the training set
+    """
+
+
+class Camelyon17Split(Enum):
+    train = 0
+    id_val = 1
+    test = 2
+    val = 3
```

Comment:
can this be `auto()`

Reply:
It could be but when the values of the enum actually have some meaning I think it's better to be safe and not rely upon the ordering to auto-generate them.